src/lib/entities/order.py

```diff
@@ -7,6 +7,8 @@ def __init__(self):
 
         self.id = None  # integer
         self.order_details = None  # list
+        self.order_status = None # enum
```

Comment:
could you rename to `status`? This is already within the `order` entity

Reply:
Or maybe `preparation_status`? up to you

Reply:
done. renamed to status.